ddtrace/tracer/sqlcomment_test.go

```diff
@@ -112,6 +146,9 @@ func TestSQLCommentCarrier(t *testing.T) {
 				root := tracer.StartSpan("service.calling.db", WithSpanID(traceID)).(*span)
 				root.SetTag(ext.SamplingPriority, tc.samplingPriority)
 				spanCtx = root.Context()
+				for k, v := range tc.spanTags {
+					root.SetTag(k, v)
+				}
```

Comment:
it seems this strategy doesn't work anymore, since we are not passing this information in span tags anymore and instead we are passing it in the `SQLCommentCarrier` struct (created in line 154 of this file).

Reply:
Extra code at this point, removing it